www/janm53/sp2/scripts/settings.js

```diff
@@ -0,0 +1,55 @@
+// Funkce pro načtení směnného kurzu z API a uložení do lokálního úložiště.
+function fetchExchangeRate(fromCurrency, toCurrency) {
+    // Dotaz na API pro získání směnného kurzu.
+    fetch(`https://www.alphavantage.co/query?function=CURRENCY_EXCHANGE_RATE&from_currency=${fromCurrency}&to_currency=${toCurrency}&apikey=Y10F3YZMEMIM7Q22`)

```

Comment:
"Invalid API call. Please retry or visit the documentation (https://www.alphavantage.co/documentation/) for CURRENCY_EXCHANGE_RATE."